resources/metadata/detectExecuteScan.yaml

```diff
@@ -257,6 +257,15 @@ spec:
           - STEPS
           - STAGES
           - PARAMETERS
+      - name: generateReportsForEmptyProjects
+        type: bool
+        default: false
+        description:
+          "If enabled, it will generate reports for empty projects. This could be useful to see the compliance reports in Sirius"
```

Comment:
a small change, could we replace SAP specefic term `Sirius `with generic term `program mangement tool` , since this in the open source